### ISSUE TITLE
level=error msg="Remotedialer proxy error" error="dial tcp 178.128.82.167:8080: i/o timeout"

### ISSUE THREAD

**Issue by GerBreOwn** [inlets/inlets#171]
when I run inlets client with the remote = "ws://167.172.216.55:8080", the token as provided, upstream = 3 different names and IP addresses I get the above error message. This is the 3rd day and I can't figure out what the problem is. For my droplet I changed from SGP1 to SFO2. I am NOT using Docker, Kubernetes nor OpenFaas
## Expected Behaviour

I should be able to connect to my Rasberrys.

## Steps to Reproduce (for bugs)

1.I created a droplet and got an IP address
2.I used that IP address in the "inlets client" remote statement
3.I entered the token I was given
4.I entered the name & local IP address for the 3 Raspberrys  
5.It said "INFO[0000] Connecting to proxy  url="ws://167.172.216.55:8080/tunnel" and just ended
## Context

I am NOT able to connect to the web sites on my Raspberrys. I am trying to accomplish connections to my 3 Raspberrys

## Your Environment

* inlets version `inlets --version` 2.7.0

* Docker/Kubernetes version `docker version` / `kubectl version`: NONE: NOT USED

* Operating System and version (e.g. Linux, Windows, MacOS):Linux Mint 19.3 on laptop, Raspian Buster on Raspberrys and Ubuntu 18.04 on DigitalOcean

* Link to your project or a code example to reproduce issue:

**Comment by GerBreOwn** [comment]
I now found a file /etc/default/inlet that had some old data in it such as remote IP, token etc. I changed that and now when I try to connect to my domains I get some wordpress first page. When I scroll to the bottom of the page it shows one of my OLD IP addresses. It shows this for all of my domains on DigitalOcean. This is some progress as I an not getting any error messages but still NOT my web sites.
